chapter_computer-vision/transposed-conv.md

```diff
@@ -82,22 +119,32 @@ tconv.initialize()
 tconv(conv(X)).shape == X.shape
 ```
 
+```{.python .input}
+#@tab pytorch
+X = torch.rand(size=(1, 10, 16, 16))
+conv = nn.Conv2d(10, 20, kernel_size=5, padding=2, stride=3)
+tconv = nn.ConvTranspose2d(20, 10, kernel_size=5, padding=2, stride=3)
+tconv(conv(X)).shape == X.shape
+```
+
 ## Analogy to Matrix Transposition
 
 The transposed convolution takes its name from the matrix transposition. In fact, convolution operations can also be achieved by matrix multiplication. In the example below, we define a $3\times 3$ input $X$ with a $2\times 2$ kernel $K$, and then use `corr2d` to compute the convolution output.
 
 ```{.python .input}
-X = np.arange(9).reshape(3, 3)
-K = np.array([[0, 1], [2, 3]])
+#@tab all
+X = d2l.arange(9.0).reshape(3, 3)
```

Comment:
Is it necessary to use 9.0 in PyTorch?

Reply:
I am merging it first. If we can use 9, just send another pr to simplify :)

Reply:
We can use 9, in that case, we'll need to cast the tensor to float32: `X.float()`. I can make the change if that's preferred?

Reply:
Then that's ok